sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentHealthcareEntitiesCollection.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Collection of <see cref="DocumentHealthcareEntities"/> objects in a document.
+    /// </summary>
+    public class DocumentHealthcareEntitiesCollection : ReadOnlyCollection<DocumentHealthcareEntities>
```

Comment:
this is going to be confusing and explains why in the sample you have to do `result.Entities.Entities`.
i.e. here you have warnings, and in `DocumentHealthcareEntities` you also have warnings, so which one should the user use?

Reply:
and in theory there are entities here and in `DocumentHealthcareEntities`